data/src/main/scala/cats/data/Or.scala

```diff
@@ -0,0 +1,186 @@
+package cats
```

Comment:
I think you might have accidentally added this back to git. It has been replaced by `Xor`.